Control.Geocoder.css

```diff
@@ -60,7 +60,7 @@
 }
 
 .leaflet-control-geocoder-throbber .leaflet-control-geocoder-icon {
-  background-image: url(images/throbber.gif);
+  background-image: url(images/throbber.svg);
```

Comment:
The `images/throbber.svg` file has not been committed?

Reply:
You're right. Now I've committed the file :slightly_smiling_face: